packages/cloud-cognitive/src/components/Datagrid/useExpandedRow.js

```diff
@@ -5,15 +5,28 @@
  * US Government Users Restricted Rights - Use, duplication or disclosure
  * restricted by GSA ADP Schedule Contract with IBM Corp.
  */
-import { useState } from 'react';
+import { layout01, layout02, layout05 } from '@carbon/themes';
```

Comment:
Rather than add these in the JS file, I think it makes more sense to add an SCSS file. You can find examples of how we do this in the package if you look in files like `useActionsColumns.js` or `useStickyColumn.js` and their respective style files (`../styles/....scss` and add to `index.scss`).